scripts/packages/pyheron/setup.py

```diff
@@ -0,0 +1,42 @@
+"""Pyheron setuptools based setup module"""
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+requirement_file = os.path.join(here, 'requirements.txt')
+with open(requirement_file, 'r') as f:
+  raw_requirements = f.read().strip()
+
+requirements = raw_requirements.split('\n')
+print "Requirements: %s" % requirements
+
+long_description = "Pyheron package allows a developer to write python topology in Heron. " \
+                   "Pyheron is backward compatible with the popular python API called " \
+                   "streamparse. It can be run in various clusters including Mesos/Aurora, " \
+                   "Mesos/Marathon, YARN, etc."
+
+setup(
+  name='pyheron',
+  version='0.0.1',
+  description='Python API for Heron topology',
+  long_description=long_description,
+
+  url='https://github.com/twitter/heron',
+
+  author='Heron Release Team',
+  author_email='heronstreaming@gmail.com',
+
+  classifiers=[
+    'Development Status :: 3 - Alpha',
+
+    'Intended Audience :: Developers',
+    'Topic :: Realtime streaming',
+
+    'Programming Language :: Python :: 2.7',
+  ],
+
+  keywords='heron topology python',
+  packages=find_packages(),
```

Comment:
we should specify license somewhere? Is this related to #1259 ?
![image](https://cloud.githubusercontent.com/assets/3656079/17682706/42e56a7a-6302-11e6-989c-2c3e114e531b.png)